MTM-56319 add contextual help info (#1510)
SoftwareAG/c8y-docs: content/web/application-configuration-bundle/application-options.md

Using the static options `dynamicOptionsUrl` the application will try to load a json from the specified URL at boot time. In the platform's built-in applications this option is set to `/apps/public/public-options/options.json` as that mechanism to provide instance level and {{< enterprise-tenant >}} customization.
As this property is defined statically at build time, it is possible for the application developer to decide if and where from their applications should load the dynamic fetched options at runtime.

> A common use-case is for example to disable the contextual help on each page, as it is only available in english. Therefore you would need to upload an application on the context-path `public-options` and add a options.json to it with the following content:

Reviewer: ```suggestion
A common use case, for example, is to disable the contextual help on each page, since it is only available in English. To do so you must upload an application on the context-path `public-options` and add an *options.json* to it with the following content:
```

```json
{
  "contextHelp": false
}
``` 

{{< c8y-admon-tip >}}

> The `public-options` application is automatically created if you have assigned a branding via the UI based branding editor. You can then simply download the zip-file, align the JSON and re-upload it again.

Reviewer: ```suggestion
The `public-options` application is automatically created if you have a branding assigned via the UI-based branding editor. You can then simply download the ZIP file, align the JSON accordingly and upload it again.
```

{{< /c8y-admon-tip >}}

> With this approach all run-time options of the application can be changed even if they are not listed in the UI based branding editor. The full list of options can be found [here](http://resources.cumulocity.com/documentation/websdk/ngx-components/classes/ApplicationOptions.html).

Reviewer: ```suggestion
With this approach all runtime options of the application can be changed even if they are not listed in the UI-based branding editor. The full list of options can be found [here](http://resources.cumulocity.com/documentation/websdk/ngx-components/classes/ApplicationOptions.html).
```
